test/common/algos.mocha.coffee

```diff
@@ -518,12 +518,12 @@ describe 'User', ->
       expect(user.items.hatchingPotions).to.eql {}
       expect(user.items.food).to.eql {}
 
-    it 'gets honey', ->
+    it 'gets red candy', ->
```

Comment:
Suggestions? It's another of those goofy things where we're testing a random output.